Tests/GetTests/ClientAuthorizationTests.swift

```diff
@@ -18,45 +18,93 @@ final class APIClientAuthorizationTests: XCTestCase {
             $0.sessionConfiguration.protocolClasses = [MockingURLProtocol.self]
         }
     }
+
+    func testAuthorizationHeaderWidhValidToken() async throws {
+        // GIVEN
+        delegate.token = Token(value: "valid-token", expiresDate: Date(timeIntervalSinceNow: 1000))
+        let url = URL(string: "https://api.github.com/user")!
+        var mock = Mock.get(url: url, json: "user")
+        mock.onRequest = { request, arguments in
+            XCTAssertEqual(request.allHTTPHeaderFields?["Authorization"], "token valid-token")
+        }
+        mock.register()
+
+        // WHEN
+        try await client.send(.get("/user"))
+    }
     
-    func testAuthorizationHeaderIsPassed() async throws {
+    func testAuthorizationHeaderWithExpiredToken() async throws {
         // GIVEN
+        delegate.token = Token(value: "expired-token", expiresDate: Date(timeIntervalSinceNow: -1000))
+        let url = URL(string: "https://api.github.com/user")!
+        var mock = Mock.get(url: url, json: "user")
+        mock.onRequest = { request, arguments in
+            XCTAssertEqual(request.allHTTPHeaderFields?["Authorization"], "token valid-token")
+        }
+        mock.register()
+        
+        // WHEN
+        try await client.send(.get("/user"))
+    }
+
+    func testAuthorizationHeaderWithInvalidToken() async throws {
+        // GIVEN
+        delegate.token = Token(value: "invalid-token", expiresDate: Date(timeIntervalSinceNow: 1000))
         let url = URL(string: "https://api.github.com/user")!
         var mock = Mock(url: url, dataType: .json, statusCode: 401, data: [
             .get: "Unauthorized".data(using: .utf8)!
         ])
-        
         mock.onRequest = { request, arguments in
-            XCTAssertEqual(request.allHTTPHeaderFields?["Authorization"], "Bearer: expired-token")
+            XCTAssertEqual(request.allHTTPHeaderFields?["Authorization"], "token invalid-token")
 
-            self.delegate.token = "valid-token"
             var mock = Mock.get(url: url, json: "user")
             mock.onRequest = { request, arguments in
-                XCTAssertEqual(request.allHTTPHeaderFields?["Authorization"], "Bearer: valid-token")
+                XCTAssertEqual(request.allHTTPHeaderFields?["Authorization"], "token valid-token")
```

Comment:
The [GitHub API](https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps#3-use-the-access-token-to-access-the-api) uses `token` instead of `Bearer`.

Also, since `:` is generally not placed immediately after the name of the authentication scheme in the Authorization header, I changed the value in the test to reduce confusion.